packages/server/tests/acceptance/estimateGasPrecompile.spec.ts

```diff
@@ -710,45 +1130,181 @@ describe('EstimatePrecompileContract tests', function () {
     isWithinDeviation(gasResult.gasUsed, estimateGasResponse, lowerPercentBound, upperPercentBound);
   });
 
+  //EGP-043 - The test is excluded due to an issue in mirror node
+  it.skip('should call estimateGas with createFungible token function', async function () {
+    let estimateContractTokenCreate = new ethers.Contract(
+      prefix + EstimatePrecompileContractAddress,
+      EstimatePrecompileContractJson.abi,
+      accounts[0].wallet,
+    );
+
+    const gasOptions = await Utils.gasOptions(requestId, 15_000_000);
+    let gasLimit = gasOptions.gasLimit;
+    let gasPrice = gasOptions.gasPrice;
+
+    const tx = await estimateContractTokenCreate.createFungibleTokenPublic.populateTransaction(
+      accounts[0].wallet.address,
+      {},
+    );
+
+    console.log('Gas Limit is ' + tx.gasLimit);
+    console.log('Gas Price is ' + tx.gasPrice);
+
+    const estimateGasResponse = await relay.call(RelayCalls.ETH_ENDPOINTS.ETH_ESTIMATE_GAS, [tx]);
+
+    const txNewToken = await estimateContractTokenCreate.createFungibleTokenPublic(
+      prefix + EstimatePrecompileContractAddress,
+      EstimatePrecompileContractJson.abi,
+      accounts[0].wallet,
+    );
+
+    const newTokenAddress = (await txNewToken.wait()).logs.filter(
+      (e) => e.fragment.name === Constants.HTS_CONTRACT_EVENTS.CreatedToken,
+    )[0].args[0];
+    console.log(newTokenAddress);
+
+    //isWithinDeviation(gasResult.gasUsed, estimateGasResponse, lowerPercentBound, upperPercentBound);
```

Comment:
We have lots of commented out code. Do we need it ?

Reply:
The commented lines are in a tests that were blocked due to an issue in the mirror node. They will be updated and the unneeded commented lines will be removed